individual_trajectories_bridge/CMakeLists.txt

```diff
@@ -19,7 +19,7 @@ find_package(ament_cmake REQUIRED)
 find_package(rclcpp REQUIRED)
 find_package(rclcpp_action REQUIRED)
 find_package(rmw_implementation_cmake REQUIRED)
-find_package(control_msgs REQUIRED)
+find_package(control_msgs 2.0.0 REQUIRED)
```

Comment:
I am not a fan of defining versions of packages within the CMake meaning we need to remember this and maintain if new version is release. The weird thing is that is isolated case as far as I know. We tested without setting the version on 3-4 different machines and did not produce any problem.

Reply:
I do agree with you. Are you using a clean machine? You can use the Docker image available for CI to test it out.

Reply:
@alesolano tested this without the setting the version in a docker and he did not have any issues. @alesolano can you confirm this?

Reply:
Yes. Tested a couple of times, last time being yesterday.

Reply:
So this is not needed, I'm going to close it.

Reply:
Although its true that we should avoid this, this does not restrict the version. It only ensures that the compiler is capturing the ros2 version of control msgs.

If we are not confortable with this we could simply add a troubleshoot note to the readme. I think this same issue will happen in more systems, not inly mine.

Reply:
Lets add troubleshoot note is easier.